tests/e2e/utils/src/components.js

```diff
@@ -352,6 +352,31 @@ const createSimpleOrder = async ( orderStatus = 'Pending payment' ) => {
 	return variablePostIdValue;
 };
 
+/**
+ * Creates a batch of orders from the given `statuses`
+ * using the "Batch Create Order" API.
+ * 
+ * @param statuses Array of order statuses
+ */
+const batchCreateOrders = async (statuses) => {
+	const defaultOrder = config.get('orders.basicPaidOrder');
+	const path = '/wc/v3/orders/batch';
+
+	// Create an order per status
+	const orders = statuses.map((s) => {
+		return {
+			...defaultOrder,
+			status: s
+		};
+	});
```

Comment:
This would be more readable if the variable was `status`:

```
const orders = statuses.map((status) => {
		return {
			...defaultOrder,
			status
		};
	});
```

Reply:
Thanks @rrennick. Fixed in https://github.com/woocommerce/woocommerce/pull/30184/commits/b116756b0967dde7fd61de6edbdcf2790a4b056c